explorer/go.mod

```
@@ -0,0 +1,237 @@
+module explorer
+
+go 1.21.0
```

Maybe remove patch so we match against the minor version:

```suggestion
go 1.21
```